examples/docs_snippets/docs_snippets/guides/components/creating-an-inline-component/generated/1-dg-scaffold-defs-inline-component.txt

```diff
@@ -0,0 +1,3 @@
+dg scaffold defs inline-component --typename AssetWithSchedule assets_with_schedules
```

Comment:
This shows up in the doc with a line break, which means it ends up formatted incorrectly when copied. Maybe add a `\`  after `AssetWithSchedule` ?